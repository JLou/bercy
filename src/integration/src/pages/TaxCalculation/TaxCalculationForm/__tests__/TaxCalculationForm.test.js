import React from 'react';
import { render } from '@testing-library/react';
import { TaxCalculationForm } from '../TaxCalculationForm';

describe('TaxCalculationFormForm', () => {
  it('Renders TaxCalculationForm component without crashing', () => {
    const { asFragment } = render(<TaxCalculationForm />);
    expect(asFragment()).toMatchSnapshot();
  });
});
