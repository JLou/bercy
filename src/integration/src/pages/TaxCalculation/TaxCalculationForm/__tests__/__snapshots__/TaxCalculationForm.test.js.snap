// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaxCalculationFormForm Renders TaxCalculationForm component without crashing 1`] = `
<DocumentFragment>
  <h1
    class="af-title--content"
  >
    Formulaire
  </h1>
  <div
    class="flex-container"
  >
    <dl>
      <dt>
        Nombre adulte :
      </dt>
      <dd>
        <input
          class="af-form__input-text"
          id="adultnumber"
          label="test"
          name="adultnumber"
          type="text"
          value=""
        />
      </dd>
    </dl>
    <dl>
      <dt>
        Montant salaire :
      </dt>
      <dd>
        <input
          class="af-form__input-text"
          id="salaryamount"
          label="test"
          name="salaryamount"
          type="text"
          value=""
        />
      </dd>
    </dl>
    <dl>
      <dt>
        Nombre enfant :
      </dt>
      <dd>
        <input
          class="af-form__input-text"
          id="numberofchildren"
          label="test"
          name="numberofchildren"
          type="text"
          value=""
        />
      </dd>
    </dl>
    <dl>
      <dt>
        Ann√©e :
      </dt>
      <dd>
        <div
          class="af-form__select-container"
        >
          <select
            class="af-form__input-select"
            name="year"
          >
            <option
              value="choice1"
            >
              2019
            </option>
            <option
              value="choice2"
            >
              2020
            </option>
          </select>
          <span
            class="glyphicon glyphicon-menu-down"
          />
        </div>
      </dd>
    </dl>
  </div>
  <div
    class="af-form__group af-form__btn"
  >
    <button
      class="btn af-btn af-btn--hasiconLeft"
      id="validation-button"
    >
      <span
        class="af-btn__text"
      >
        Calculer
      </span>
      <i
        class="glyphicon glyphicon-stats"
      />
    </button>
  </div>
</DocumentFragment>
`;
